spec.bs

```diff
@@ -376,22 +376,23 @@ To enroll a user for Secure Payment Confirmation, relying parties should call
 {{AuthenticationExtensionsClientInputs/payment}} [=WebAuthn Extension=]
 specified.
 
-Note: We currently require an extension in order to allow credential creation
-      in an iframe and to allow the browser to cache SPC credentials in the
-      absence of [=WebAuthn Conditional UI=]. Future versions of this
-      specification may remove the requirement for the extension.
+Note: We currently require an extension in order to allow cross-origin
+      credential creation in an iframe and to allow the browser to cache
+      SPC credentials in the absence of [=WebAuthn Conditional UI=].
+      Future versions of this specification may remove the requirement
+      for the extension.
 
 # Authentication # {#sctn-authentication}
 
 To authenticate a payment via Secure Payment Confirmation, this specification
 defines a new [=payment method=], "[=secure-payment-confirmation=]". This
 payment method confirms the transaction with the user and then performs an
 [=authentication ceremony=] to authenticate the user and create a signed blob
-representing the transaction.
+representing the payment-specific information.
 
 At a high level, authentication for Secure Payment Confirmation is similar to
 [[webauthn-3]], with one major conceptual shift. Secure Payment Confirmation
-allows a third-party (e.g. the merchant) to trigger an authentication ceremony
+allows a third-party (e.g., the merchant) to trigger an authentication ceremony
```

Comment:
So TIL that American English requires a comma after `e.g.`... British English explicitly does not allow it. Fun!